### PR TITLE
#Issue 12427 Update Adcp Parser

### PR DESCRIPTION
**Pull request by vipullakhani**

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,3 +1,18 @@
+# Version 0.5.9
+
+* Issue #12427 Update ADCP Parsers and Consolidate ADCP stream:
```
Comment (danmergens) [inline RELEASE_NOTES.md]: streams

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,3 +1,18 @@
...
+    * stream "adcp_velocity_earth" replaces:
```
Comment (danmergens) [inline RELEASE_NOTES.md]: Markdown supports formatting such as `adcp_velocity_earth` to denote variable names. Why are you highlighting `adcp_velocity_earth` differently than `adcp_velocity_glider`?

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -1,3 +1,18 @@
...
+        * adcp_velocity_inst
+    * stream "adcp_velocity_beam" replaces:
+        * adcp_pd0_beam_parsed
+        * vadcp_5thbeam_pd0_beam_parsed
+        * vadcp_pd0_beam_parsed
+    * stream "adcp_system_configuration" replaces:
+        * vadcp_4beam_system_configuration
+    * stream "adcp_system_configuration_5" replaces:
+        * vadcp_5thbeam_system_configuration
+
```
Comment (danmergens) [inline RELEASE_NOTES.md]: We probably should identify all the parsers that have been updated so the Data Team will know which parsers need to be rerun.

File: RELEASE_NOTES.md (hunk 2 above)
Comment (vipullakhani) [reply RELEASE_NOTES.md]: becuase adcp_velocity_earth is new stream

File: RELEASE_NOTES.md (hunk 3 above)
Comment (vipullakhani) [reply RELEASE_NOTES.md]: its already attached to the ticket, like which parser should rerun

File: RELEASE_NOTES.md (hunk 2 above)
Comment (danmergens) [reply RELEASE_NOTES.md]: Quotation marks indicate one of two things:
* direct quotation; she said, "I like spam."
* exact match; you have to spell it "hyperbole."
* facetious; the air quotes, we are going to Long Island Mike's for "salad" and "soda."

I recommend using other formatting to delineate it from other streams, such as bold font.

File: RELEASE_NOTES.md (hunk 3 above)
Comment (danmergens) [reply RELEASE_NOTES.md]: As a courtesy to the Data Team, that will be looking at the RELEASE_NOTES.md file, not the pull request, can you please add in the affected streams.

**Pull request merged**